media/js/firefox/whatsnew/whatsnew-75.js

```diff
@@ -19,6 +19,15 @@ if (typeof window.Mozilla === 'undefined') {
                 document.querySelector('.c-page-header').classList.add('show-up-to-date-message');
             }
         });
+
+        client.getFxaDetails(function(details) {
+            if (details.setup && details.browserServices.sync.mobileDevices > 0) {
```

Comment:
Detecting synced devices isn't currently part of our standardized mozilla-fxa.js lib, but I wonder if we should add it there?

Reply:
Could be worth adding yeah, file a followup issue perhaps?